src/BloomBrowserUI/bookEdit/copyrightAndLicense/CopyrightPanel.tsx

```diff
@@ -187,6 +187,10 @@ export const CopyrightPanel: React.FunctionComponent<{
                                 p {
                                     margin-block-end: 0 !important; // override DialogMiddle setting and browser default
                                 }
+                                // Somehow just putting a color rule on PWithLink doesn't work, so we put it here.
+                                .translated a {
```

Comment:
Why did you include the `.translated` class? Seems like it shouldn't be necessary. But worse, what if it isn't translated?